advanced-creation.md

```diff
@@ -111,3 +111,112 @@ const unicorn = got.create(settings);
 // Same as:
 const unicorn = got.extend({headers: {unicorn: 'rainbow'}});
 ```
+
+### Merging instances
+
+#### got.mergeInstances(instanceA, instanceB, ..., [methods])
+
+Merges many instances into a single one:
+- options are merged using [`got.mergeOptions()`](readme.md#gotmergeoptionsparentoptions-newoptions) (+ hooks are merged too),
+- handlers are stored in an array.
+
+##### [instances](readme.md#instances)
+
+##### [[methods]](#methods)
+
+Default: `instances[0].defaults.methods`
+
+#### Usage
+
+```js
+const got = require('./got');
+
+// #1 You can deny redirects that lead to other sites than specified
```

Comment:
When I asked for example use-cases, I really just meant a text list, but this is much better. Can you move the different examples into separate code-blocks with the comment as a header instead.

Reply:
Sure! :)